### PR TITLE
Configure shared StyleCop ruleset and implement most suggestions

### PR DESCRIPTION
**Pull request by RKivioja**

### CODE REVIEW

**Comment by RKivioja** [comment]
Waits for the "devsupport" target change from "net6.0" to "netstandard2.0".

**Comment by RKivioja** [comment]
@eXpl0it3r This should be fine now.

File: source/Jobbr.Server.UnitTests/Jobbr.Server.UnitTests.csproj [hunk 1]
```diff
@@ -1,19 +1,21 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
-        <TargetFramework>net6.0</TargetFramework>
+        <TargetFrameworks>net6.0</TargetFrameworks>
```
Comment (eXpl0it3r) [inline source/Jobbr.Server.UnitTests/Jobbr.Server.UnitTests.csproj:3]: Should be singular, as we don't multi-target

File: source/Jobbr.Server/Scheduling/DefaultScheduler.cs [hunk 2]
```diff
@@ -20,9 +20,9 @@ public class DefaultScheduler : IJobScheduler
         private readonly IScheduledJobRunPlaner _scheduledJobRunPlanner;
         private readonly IRecurringJobRunPlaner _recurringJobRunPlanner;
         private readonly DefaultSchedulerConfiguration _schedulerConfiguration;
-        private readonly object _evaluateTriggersLock = new ();
+        private readonly object _evaluateTriggersLock = new object();
 
-        private List<ScheduledPlanItem> _currentPlan = new ();
+        private List<ScheduledPlanItem> _currentPlan = new List<ScheduledPlanItem>();
```
Comment (eXpl0it3r) [inline source/Jobbr.Server/Scheduling/DefaultScheduler.cs:25]: The previous "default" initialization would also be okay for me

**Review by eXpl0it3r — CHANGES_REQUESTED** [review]

**Review by eXpl0it3r — APPROVED** [review]

**Pull request merged**